content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management.rst

```diff
@@ -1,18 +1,144 @@
-:nosearch:
+:show-content:
+:hide-page-toc:
 
 ====================
 Inventory management
 ====================
 
+In the Odoo *Inventory* app, :doc:`warehouses <inventory_management/warehouses>` handle the broader
```

Comment:
This doc is well-written and organized. Everything is clearly explained, and the structure makes sense. Nice job 👍